admin/react/includes/class-custom-fonts-admin-ajax.php

```diff
@@ -0,0 +1,297 @@
+<?php
+/**
+ * Ajax class for font settings.
+ *
+ * @package     Bsf_Custom_Fonts
+ * @since       x.x.x
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * AJAX events class to manage dashboard application requests.
+ *
+ * @since x.x.x
+ */
+class BSF_Custom_Fonts_Admin_Ajax {
+	/**
+	 * Instance
+	 *
+	 * @access private
+	 * @var null $instance
+	 * @since 4.0.0
+	 */
+	private static $instance;
+
+	/**
+	 * Initiator
+	 *
+	 * @since 4.0.0
+	 * @return object initialized object of class.
+	 */
+	public static function get_instance() {
+		if ( ! isset( self::$instance ) ) {
+			self::$instance = new self();
+		}
+		return self::$instance;
+	}
+
+	/**
+	 * Errors class instance.
+	 *
+	 * @var array
+	 * @since 4.0.0
+	 */
+	private $errors = array();
+
+	/**
+	 * Constructor
+	 *
+	 * @since 4.0.0
+	 */
+	public function __construct() {
+		$this->errors = array(
+			'permission' => __( 'Sorry, you are not allowed to do this operation.', 'astra' ),
+			'nonce'      => __( 'Nonce validation failed', 'astra' ),
+			'default'    => __( 'Sorry, something went wrong.', 'astra' ),
+			'invalid'    => __( 'No post data found!', 'astra' ),
+		);
```

Comment:
Should all this be 'custom-fonts ' rather then 'astra' @imnavanath

Reply:
Updated